### PR TITLE
Updated StandardPipeline.setup() to set dictionary_path

### PR DESCRIPTION
**Pull request by bevinduckett**
`IntegrationTestApi.get_deployment()` uses the `dictionary_path` property of self.pipeline, so `StandardPipeline.setup()`  needs to set that value

| | |
|:---|:---|
|**_Related Issue(s)_**|  none |
|**_Has Unit Tests (y/n)_**|  no |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

Update StandardPipeline.setup()  to set the object's `dictionary_path` field 

## Rationale

Without this change, `IntegrationTestApi.get_deployment()` always fails 

## Testing/Review Recommendations

Add `get_deployment()` to any test using `fprime_test_api`, without this change it will fail with `Error: File not found at path: None`. With this change it'll work 

## Future Work

Note any additional work that will be done relating to this issue.

### CODE REVIEW

**Comment by thomas-bc** [comment]
> Without this change, IntegrationTestApi.get_deployment() always fails

@kevin-f-ortega is this function not actively being used?

File: src/fprime_gds/common/pipeline/standard.py [hunk 1]
```diff
@@ -90,6 +90,7 @@ def setup(
                 f"{file_store} is not writable. Fix permissions or change storage directory with --file-storage-directory."
             )
         self.__dictionaries = dictionaries
+        self.dictionary_path = getattr(self.dictionaries,"_dictionary_path")
```
Comment (thomas-bc) [inline src/fprime_gds/common/pipeline/standard.py]: This should work as well since it's defined as a `@property`, no?
```suggestion
        self.dictionary_path = self.dictionaries.dictionary_path
```

Comment (thomas-bc) [reply src/fprime_gds/common/pipeline/standard.py]: Technically I'd actually argue the bug is that `dictionary_path` shouldn't be a property of StandardPipeline at all, so the cleanest would be to fix `get_deployment()` function to use:

```diff
+ dictionary = str(self.pipeline.dictionaries.dictionary_path)
- dictionary = str(self.pipeline.dictionary_path)
```

No reason to unnecessarily copy the `dictionary_path` up into StandardPipeline since we already have a reference to the Dictionaries object, which holds `dictionary_path`

Comment (bevinduckett) [reply src/fprime_gds/common/pipeline/standard.py]: What tipped me towards setting `self.dictionary_path` in `StandardPipeline.setup()` was that `dictionary_path` is defined as one of the attributes of the object in `StandardPipeline.__init__()` and the effort to verify the code-base wide implications of removing that attribute is higher (i.e. setting the attribute/property to a valid value is fully backwards compatible, removing it is not) 
What do you think of instead removing the dictionary_path attribute & adding the below property function to StandardPipeline? 
```Python 
    @property
    def dictionary_path(self):
        """Dictionaries member"""
        return self.dictionaries.dictionary_path if self.dictionaries else None
```

Comment (thomas-bc) [reply src/fprime_gds/common/pipeline/standard.py]: @bevinduckett I like that idea a lot!

Comment (bevinduckett) [reply src/fprime_gds/common/pipeline/standard.py]: Committed discussion change

**Review by thomas-bc — DISMISSED** [review]
LGTM.

@kevin-f-ortega please confirm when you get back, I would have expected for example https://github.com/nasa/fprime/blob/a9c3726c451a1530bd3e8c71018192976f4506ca/Svc/CmdSequencer/test/int/test_cmd_sequencer.py#L127 to be running in CI. Is it not the case?

EDIT: Ah - it seems the function is being called, but the result is not being used (only printed out)

File: src/fprime_gds/common/pipeline/standard.py [hunk 2]
```diff
@@ -90,6 +89,7 @@ def setup(
...
```
Comment (thomas-bc) [inline src/fprime_gds/common/pipeline/standard.py]: I think removing this line should solve CI error

**Review by thomas-bc — APPROVED** [review]
